atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/cli/command/RunTestsCommand.java

```diff
@@ -15,26 +15,95 @@
  */
 package com.palantir.atlasdb.performance.cli.command;
 
+import java.util.List;
+import java.util.Set;
 import java.util.concurrent.Callable;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
 
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Lists;
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
 import com.palantir.atlasdb.performance.cli.backend.PhysicalStore;
 import com.palantir.atlasdb.performance.cli.backend.PhysicalStoreType;
+import com.palantir.atlasdb.performance.test.api.PerformanceTest;
+import com.palantir.atlasdb.performance.test.api.ValueGenerator;
+import com.palantir.atlasdb.performance.test.api.annotation.PerfTest;
+import com.palantir.atlasdb.performance.test.generator.RandomValueGenerator;
 
+import io.airlift.airline.Arguments;
 import io.airlift.airline.Command;
 import io.airlift.airline.Option;
 
 @Command(name = "run", description = "Run tests")
 public class RunTestsCommand implements Callable<Integer> {
 
+    @Arguments(title = "TESTS",
+            description = "tests to run",
+            required = false)
+    private String tests;
+
     @Option(name = {"-b", "--backend"},
             title = "PHYSICAL STORE",
             description = "underyling physical store (e.g. POSTGRES)",
             required = true)
     private PhysicalStoreType type;
 
+    private static final Set<Class<? extends PerformanceTest>> ALL_TESTS =
+            ImmutableSet.<Class<? extends PerformanceTest>>builder()
+                    .build();
+
     @Override
     public Integer call() throws Exception {
-        PhysicalStore.create(type).connect();
+        KeyValueService kvs = PhysicalStore.create(type).connect();
```

Comment:
nit: It's a bit weird that we have a class called PhysicalStore that 'creates' a KeyValue Server. If I were to be puritanical, I'd say this statement should look more like: `KeyValueServices.createWith(type).connect()`